src/tests/integration/connection/connectionUpdateIdentityKeys.spec.ts

```diff
@@ -45,10 +45,22 @@ describe('Connection Update Identity Keys', () => {
   let targetUserId: string;
   let pSdk: PillarSdk;
 
-  const responseData = {
-    result: 'success',
-    message: 'Updated connections',
-  };
+  const responseData = [
+    {
+      sourceConnection: {
+        userId: 'userId',
+        targetUserId: 'target-user-id',
```

Comment:
the connection doesn't have targetUserId, the response should be { sourceConnection, targetConnection }.